res/values-ku/strings.xml

```diff
@@ -1310,7 +1310,7 @@ FERMAN:
     <string name="You_are_already_friends_">Hûn jixwe heval in.</string>
     <string name="Account_must_be_1_day_old_to_send_mail_">Ji bo ku em bikarin nameyan bişînin divê hesab herî kêm 1 rojî be.</string>
     <string name="Translator">Wergêr</string>
-    <string name="SUCCESS">SUCCESS</string>
+    <string name="SUCCESS">serketinî</string>
```

Comment:
Please preserve 100% of the capital letters here (SERKETİNI)

Reply:
Should I modify it or open a new request?

Reply:
Modify it and send in this same PR, no need to create another ^^

Reply:
I can't edit it it's locked

Reply:
I can't edit